microsite/sidebars.json

```diff
@@ -107,7 +108,7 @@
       {
         "type": "subcategory",
         "label": "AWS S3",
-        "ids": ["integrations/aws-s3/locations"]
+        "ids": ["integrations/aws-s3/locations", "integrations/aws-s3/discovery"]
```

Comment:
prettier complains because these are on one line instead of two and there's no newline at the end of the file.

From the project root:

```shell
cd microsite
yarn install
yarn prettier --write sidebars.json
```

Reply:
Ahh, didn't run `yarn install` before `prettier` looks like this resolved it.